psh/test-mkdir.py

```diff
@@ -102,7 +104,8 @@ def assert_first_prompt(p):
 def harness(p):
     chars = list(set(string.printable) - set(string.whitespace) - set('/'))
 
-    assert_first_prompt(p)
+    run_psh(p)
```

Comment:
please note that currently running `psh` from `psh` does nothing (this is a separate issue). I'm writing it because when it will start spawning subshell we should exit from psh at the end of each test (or do we assume we will always reboot between the tests?)

No changes currently needed, just keep it in mind.